docs/security/users-and-teams/auditing/index.md

```diff
@@ -96,3 +96,9 @@ Deleting the archived files will permanently erase the audit entries. As a safeg
 :::
 
 The archived files can also be accessed via the Octopus REST API endpoints `/api/events/archives` and `/api/events/archives/{filename}`.
+
+## IP address forwarding
+
+From **Octopus 2023.1**, the originating IP address of a request is recorded as part of any audit event. If you host Octopus on-premises and run multiple nodes in a High Availability setup, incoming requests will be redirected from your load balancer. This means that by default, the IP address recorded with any event will be the IP address of your load balancer. To resolve this, you can configure any trusted network addresses via **{{Configuration, Settings, Web Portal, Trusted Proxies}}**.
+
+Octopus reads forwarded IP addresses from the `X-Forwarded-For` header. Given the IP address of the client sending the request is trusted, the rightmost IP address that is **not** included in the list of trusted proxies will be used as the IP address for the event. If all IP addresses are trusted, the leftmost value in the `X-Forwarded-For` header will be used. For example, if the network address `0.0.0.0/0` is included as a trusted proxy, then any request will always use the leftmost IP address found in the `X-Forwarded-For` header.
```

Comment:
I could see this getting fleshed out a little bit more to provide more clarity to the rules, but im also happy enough with what you have here for the moment

Reply:
I expanded upon the rules to make it a bit clearer.